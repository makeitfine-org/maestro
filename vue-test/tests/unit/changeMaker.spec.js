import changeMaker from './changeMaker';

it('gives proper change.', () => {
    expect(changeMaker(1.99)).toMatchSnapshot();
    expect(changeMaker(2.75)).toMatchSnapshot();
    expect(changeMaker(0.24)).toMatchSnapshot();
    // And so on
});