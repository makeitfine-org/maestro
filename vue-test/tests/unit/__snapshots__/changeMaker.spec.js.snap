// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gives proper change. 1`] = `
Object {
  "dimes": 2,
  "nickels": 0,
  "pennies": 4,
  "quarters": 7,
}
`;

exports[`gives proper change. 2`] = `
Object {
  "dimes": 0,
  "nickels": 0,
  "pennies": 0,
  "quarters": 11,
}
`;

exports[`gives proper change. 3`] = `
Object {
  "dimes": 2,
  "nickels": 0,
  "pennies": 4,
  "quarters": 0,
}
`;
